bin/update_pythons.py

```diff
@@ -115,8 +115,8 @@ def __init__(self, arch_str: ArchStr):
         self.arch = arch_str
 
     def update_version_windows(self, spec: Specifier) -> ConfigWinCP:
```

Comment:
This'll probably take some more changes. But I'm guessing we'll have to wait for a final release of 7.3.4 to fix it.